### PR TITLE
Bugfix/7835/fix poll results accessibility

### LINKED ISSUE #7835: Poll result details screen is not keyboard navigatable
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

## Steps to reproduce
1. Create a poll
2. Vote
3. End poll
4. Open result screen
5. Click on any text
6. Keyboard navigate to the avatar block to expand the list
7. Tab one more time to enter the popup

### Expected behaviour
Can navigate back and close the popup and move to the next one

### Actual behaviour
Tabbing breaks

---

## Remarks
- [ ] Either the NcPopoveris still/completely broken
- [ ] Or it breaks because button is used instead of NcButton

### PR DESCRIPTION
**Pull request by marcoambrosini**
fix #7835 

https://user-images.githubusercontent.com/26852655/196631018-c31bce14-4d03-467b-83c7-683cd0ed9295.mov

### CODE REVIEW

**Comment by nickvergessen** [comment]
Contradicting signals, ready to review (label) or a draft (state)

**Comment by nickvergessen** [comment]
Also wrong milestone

File: src/components/MessagesList/MessagesGroup/Message/MessagePart/PollVotersDetails.vue [hunk 1]
```diff
@@ -20,7 +20,8 @@
 -->
 
 <template>
-	<NcPopover trigger="hover">
+	<NcPopover trigger="hover"
+		:focus-trap="false">
```
Comment (nickvergessen) [inline src/components/MessagesList/MessagesGroup/Message/MessagePart/PollVotersDetails.vue:24]: this is very cumbersome.... After "Show results" I have to tab myself through all the remaining UI including the participant listing before I reach the popup to find the poll voting details

Comment (nickvergessen) [reply src/components/MessagesList/MessagesGroup/Message/MessagePart/PollVotersDetails.vue:24]: But that seems also the case on master, is it not this?
Is the container missing on the Poll Modal?

**Review by nickvergessen — CHANGES_REQUESTED** [review]

File: src/components/MessagesList/MessagesGroup/Message/MessagePart/PollVotersDetails.vue (hunk 1 above)
Comment (marcoambrosini) [reply src/components/MessagesList/MessagesGroup/Message/MessagePart/PollVotersDetails.vue:24]: I can't reproduce this neither on chrome nor firefox. As soon as i hit enter on see result I'm in the dialog with the focus on the "pollVotersDetails" component. Could make sure you pulled and test again pleez?

Comment (nickvergessen) [reply src/components/MessagesList/MessagesGroup/Message/MessagePart/PollVotersDetails.vue:24]: Hmm seems to work on this PR.

After rebasing to master (was lacking 100 commits behind) it does work as I described. Might be the broken focus trap thing in 7.0.1+

So I'd say let's wait for a new vue lib version and then revisit the PR?

Comment (marcoambrosini) [reply src/components/MessagesList/MessagesGroup/Message/MessagePart/PollVotersDetails.vue:24]: ok sounds good!

**Comment by marcoambrosini** [comment]
@Antreesy or @DorraJaouad could you take over?

**Comment by nickvergessen** [comment]
Someone will take over, when the item is a priority again.

**Pull request closed without merging**